scilpy/io/image.py

```diff
@@ -23,3 +24,68 @@ def assert_same_resolution(images):
         aff_2, shape_2, _, _ = get_reference_info(i)
         if not (shape_1 == shape_2) and (aff_1 == aff_2).any():
             raise Exception("Images are not of the same resolution/affine")
+
+
+def get_data_as_mask(parser, in_img):
```

Comment:
I would not use a parser.error(), but rather a `raise IOError` or something since this function will be used pretty much everywhere in the code and not just in the `main()` of a script.